### ISSUE TITLE
[BUG] "Default Keep ID" setting not functioning in core Import All dialog

### ISSUE THREAD

**Issue by sharkbruhaha** [earlSt1/vtt-compendium-folders#196]
Setting is not working:
![image](https://user-images.githubusercontent.com/44031572/216199642-a9891e59-dd4e-40ad-a953-aefb96ae66bf.png)

```
Foundry Virtual Tabletop: Version 10, 10.291
Game System: dnd5e, 2.1.4
Active Modules: 21
Performance Mode: 2

OS: Unknown
Client: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
GPU: ANGLE (NVIDIA, NVIDIA GeForce RTX 3090 Direct3D11 vs_5_0 ps_5_0, D3D11)
Max Texture Size: 16384

No viewed scene

Actors: 0 | Items: 0 | Journal: 0 | Rollable Tables: 0 |
Playlists: 0 | Compendium Packs: 25 | Chat Messages: 2
```

**Comment by earlSt1** [comment]
Hi, was about to look into this but this looks like the core foundry UI for importing all contents of a compendium to a folder. I'll see about wrapping that so its set by the module in the next release

**Comment by earlSt1** [comment]
Fixed in v2.5.7
